src/3d/qgslayoutitem3dmap.cpp

```diff
@@ -23,6 +23,7 @@
 #include "qgslayoutitemregistry.h"
 #include "qgsoffscreen3dengine.h"
 
+#include <Qt3DRender/QCamera>
```

Comment:
probably this include is not needed if there's no other code change in this file?